.github/workflows/nuget-audit.yml

```
@@ -0,0 +1,24 @@
+name: NuGet Audit
+on:
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+defaults:
+  run:
+    shell: bash
+jobs:
+  build:
+    name: Linux
+    runs-on: ubuntu-22.04    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v4.1.0
+        with:
+          global-json-file: global.json
+      - name: Restore
+        id: restore
+        run: dotnet restore src -v m
```

Why do we need to set the verbosity here? `m` is the default anyway.

```suggestion
        run: dotnet restore src
```